services/ui-src/src/containers/UserPage.js

```diff
@@ -161,7 +161,7 @@ const UserPage = () => {
   const { userProfile, setUserInfo, updatePhoneNumber, userRole, userStatus } =
     useAppContext();
   const location = useLocation();
-  const { userId } = useParams() ?? {};
+  let { userId } = useParams() ?? {};
```

Comment:
curious why this change was made........might even be flagged by lint if setting with let but is never overridden.....but also probably userId is something we dont want to let get overwritten?

Reply:
you're right I did this when I was considering how to encrypt the email/userId before Andy fixed it